strategies/affected/src/main/java/org/arquillian/smart/testing/strategies/affected/ClassFileIndex.java

```diff
@@ -46,42 +47,39 @@
 public class ClassFileIndex {
 
     private Logger logger = Logger.getLogger(ClassFileIndex.class);
+    private static final Filter javaJdkFilter = new Filter("", "java*");
```

Comment:
This is minor, but `JDK` stands for `Java Development Kit` ;) Maybe `coreJavaFilter` would be more appropriate here?